modules/consensus/applytransaction.go

```diff
@@ -26,13 +26,22 @@ var (
 func (cs *ConsensusSet) applySiacoinInputs(scoBucket *bolt.Bucket, pb *processedBlock, t types.Transaction) error {
 	// Remove all siacoin inputs from the unspent siacoin outputs list.
 	for _, sci := range t.SiacoinInputs {
+		scoBytes := scoBucket.Get(sci.ParentID[:])
+		if scoBytes == nil {
+			panic("nil sco bytes")
+		}
+		var sco types.SiacoinOutput
+		err := encoding.Unmarshal(scoBytes, &sco)
+		if err != nil {
+			return err
+		}
```

Comment:
I am worried that this is out of place. Perhaps this should be moved in to a `getSiacoinOutput` function (which doesn't exist at the moment).